fenix/app/src/main/java/org/mozilla/fenix/debugsettings/ui/DebugDrawer.kt

```diff
@@ -6,31 +6,118 @@ package org.mozilla.fenix.debugsettings.ui
 
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.material.Snackbar
+import androidx.compose.material.SnackbarHost
+import androidx.compose.material.SnackbarHostState
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.launch
+import mozilla.components.support.ktx.android.content.appName
+import mozilla.components.support.ktx.android.content.appVersionName
+import org.mozilla.fenix.compose.Divider
 import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.compose.inComposePreview
+import org.mozilla.fenix.compose.list.TextListItem
 import org.mozilla.fenix.theme.FirefoxTheme
 
 /**
  * The debug drawer UI.
  */
 @Composable
-fun DebugDrawer() {
-    Box(
+fun DebugDrawer(
+    navigationItems: List<String>,
+    onNavigationItemClick: (String) -> Unit,
+) {
+    val lazyListState = rememberLazyListState()
+
+    val appName: String
+    val appVersion: String
```

Comment:
Git revision, A-C version, and geckoview version may all be useful things to add here as well

Reply:
Or in a details subpage

Reply:
I'm thinking this will be a subpage. If I include Gecko version, etc., it'll be specific to browsers and not aligned with the generic use case we're working towards.